docs/mastering-plone/features.md

```diff
@@ -85,19 +85,22 @@ Opening `http://localhost:3000`, you are facing the front page of your Plone sit
 
 You can stop the frontend any time using {kbd}`ctrl c`.
 
-While developing it is not necessary to restart the frontend unless you are adding a new file.
+While developing it's not necessary to restart the frontend unless you are adding a new file.
+
+Login to your new site with "admin" and password "secret".
+
+```{figure} _static/frontpage_volto_logged_in.png
+```
 
 
 ### Change ports
 
 #### backend
 
-Now let's say you want Plone to listen on port 9080 instead of the default 8080.
-Open the file `backend/instance.yml` in your favorite editor.
-How could you do this?
+If you want Plone to listen on port 9080 instead of the default 8080, open the file `backend/instance.yml` in your favorite editor.
```

Comment:
MyST syntax for semantic markup.

```suggestion
If you want Plone to listen on port 9080 instead of the default 8080, open the file {file}`backend/instance.yml` in your favorite editor.
```